### ISSUE TITLE
Merges have change release states

### ISSUE THREAD

**Issue by langner** [cclib/cclib#294]
There has been at least one merge (#273) which changed the release state of v1.4.1, so in the future it'll be a better idea to in fact keep release in separate branches. For 2-3 of the most recent releases I'll create branches soon and revery any merges that took place after the relase.

**Comment by langner** [comment]
Version 1.4 already has a release branch, so I did it just for v1.4.1.

**Comment by berquist** [comment]
I guess I'm a little confused as to what happened at the end of https://github.com/cclib/cclib/pull/273 (something was still merged?). Is this just so we can keep adding new features/fixes in master, and cherry-pick the ones we want for the release?

**Comment by langner** [comment]
Two things happened:
1) I merged Bing's PR into master, which contained commits from before 1.4.1 was tagged. Because the tag was on master, those commits got included into the release as published on GitHub retroactively (the pypi release wasn't affected of course).
2) I reverted that merge and pushed it to master by accident and didn't fix that for about 24h.

I think now everything is in order. Master contains the merge, and the `release-1.4.1` branch has the reverted history and the `v1.4.1` tag the release uses. If anything seems still odd, let me know.
